thanks-bot/bot.js

```diff
@@ -1,35 +1,57 @@
-var Discord = require('discord.io');
-var logger = require('winston');
-var auth = require('./auth.json');
+const Discord = require('discord.js')
+const logger = require('winston')
+const auth = require('../auth.json')
 
 // Configure logger settings
-logger.remove(logger.transports.Console);
+logger.remove(logger.transports.Console)
 logger.add(logger.transports.Console, {
-    colorize: true
-});
-
-logger.level = 'debug';
+  colorize: true
+})
+logger.level = 'debug'
 
 // Initialize Discord Bot
-var bot = new Discord.Client({
-   token: auth.token,
-   autorun: true,
-});
+const bot = new Discord.Client()
```

Comment:
I saw everyone called it a client, so let's call it client too
```suggestion
const client = new Discord.Client()
```